sdk/lib/convert/json.dart

```diff
@@ -91,6 +107,31 @@ String jsonEncode(Object? object,
 ///
 /// Shorthand for `json.decode`. Useful if a local variable shadows the global
 /// [json] constant.
+///
+/// Example:
+/// ```dart
+/// final sampleJsonString =
+///     '{"text":"foo", "value":1, "status": false, "extra":null}';
+///
+/// final data = jsonDecode(sampleJsonString);
+/// print(data['text']); // foo
+/// print(data['value']); // 1
+/// print(data['status']); // false
+/// print(data['extra']); // null
+///
+/// const jsonArray = '[{"text": "foo", "value": 1, "status": true},'
+///    '{"text": "bar", "value": 2, "status":false}]';
+///
+/// final List<dynamic> dataList = jsonDecode(jsonArray);
+/// print(dataList[0]); // {text: foo, value: 1, status: true}
+/// print(dataList[1]); // {text: bar, value: 2, status: false}
+///
+/// final item = dataList[0];
+/// print(item['text']); //  foo
+/// print(item['value']); // 1
+/// print(item['status']); //false
```

Comment:
Nit: extra space before foo, and no space before false

Reply:
fixed